launcher/src/backend/ValidatorAccountManager.js

```diff
@@ -228,4 +228,42 @@ export class ValidatorAccountManager {
         return URL
     }
 
+    async setGraffitis(graffiti){
```

Comment:
it's possible to run mulitple validator services

Reply:
done 👍🏼